tutorials/promscale-run-queries.md

```diff
@@ -0,0 +1,279 @@
+## 4. Running queries using Promscale
+
+Promscale offers the combined power of PromQL and SQL, enabling you to ask any 
+question, create any dashboard, and achieve greater visibility into the systems 
+you monitor.
+
+In the configuration used in Part 3 above, Prometheus will scrape the Node Exporter every 10s and metrics will be stored in both Prometheus and TimescaleDB, via Promscale. 
+
+This section will illustrate how to run simple and complex SQL queries against Promscale, as well as queries in PromQL.
+
+### 4.1 SQL Queries in Promscale [](sql-queries)
+
+You can query Promscale in SQL from your favorite favorite SQL tool or using psql: 
+
+```bash
+docker exec -it timescaledb psql postgres postgres
+```
+
+The above command first enters our timescaledb docker container (from Step 3.1 above) and creates an interactive terminal to it. Then it opens up`psql`, a terminal-based front end to PostgreSQL (More information on psql -- [psql-docs][].
+
+Once inside, we can now run SQL queries and explore the metrics collected by Prometheus and Promscale
+
+#### 4.1.1 Querying a metric:
```

Comment:
I'd add anchor links to each of these. I can see an opportunity to link directly to one of these in a blog post of Twitter/Slack response

Reply:
And by "these", I mean each of the 4.1.x examples below... :-)

Reply:
Done!